canonical-web-and-design/jaas-dashboard
# Display the application version from the package.json

## Done

Display the application version in the sidebar, fetched from the package.json on build.

## QA

- Pull code
- Run `./run clean && ./run serve`
- Open http://0.0.0.0:8036/
- In the sidebar you should see the same version in the package.json.

## Details

Fixes #447 

## Screenshots

![Screen Shot 2020-04-23 at 2 50 18 PM](https://user-images.githubusercontent.com/532033/80148104-c4eebb80-8571-11ea-9c87-067b0049cf81.png)
